docs/en/observability/application-logs.asciidoc

```diff
@@ -1,8 +1,24 @@
 [[application-logs]]
-= Monitor application logs
+= Application logs
 
 Application logs provide valuable insight into events that have occurred within your services and applications.
 
+The ingestion of those logs requires to deal with two aspects:
+
+- <<log-types,log format>>: plaintext or ECS (structured JSON) logs.
+- <<ingest-application-logs,logs ingestion>>: how those logs are sent and written to Elasticsearch using a tool like {filebeat}.
```

Comment:
Instead of ingestion requiring format and ingestion, what about monitoring requiring format and ingestion?

```suggestion
There are two aspects to consider when monitoring application logs:

- <<log-types,log format>>: are your logs plaintext or Elastic Common Schema (ECS) logs (structured JSON)
- <<ingest-application-logs,log ingestion>>: the tool you use to send and write logs to Elasticsearch
```